tests/foreman/cli/test_template.py

```diff
@@ -95,7 +95,7 @@ def test_create_template_with_org(self):
 
     @run_only_on('sat')
     @tier2
-    def test_add_operating_system_1(self):
+    def test_positive_add_os_by_id(self):
         """@Test: Check if Template can be assigned operating system
```

Comment:
I know it is out of scope, but I can't pass by:
`Check if operating system can be assigned to template by id`